table/tables/partition_test.go

```diff
@@ -306,3 +306,69 @@ func (ts *testSuite) TestLocateRangePartitionErr(c *C) {
 	_, err := tk.Exec("INSERT INTO t_month_data_monitor VALUES (4, '2019-04-04')")
 	c.Assert(table.ErrNoPartitionForGivenValue.Equal(err), IsTrue)
 }
+
+func (ts *testSuite) TestTimeZoneChange(c *C) {
+	tk := testkit.NewTestKitWithInit(c, ts.store)
+	tk.MustExec("use test")
+	createTable := `CREATE TABLE timezone_test (
```

Comment:
How about removing some redundant columns in the test case? It will make the case tidy.

Reply:
Done